homeassistant/components/lutron_caseta/__init__.py

```diff
@@ -94,6 +96,21 @@ def name(self):
         """Return the name of the device."""
         return self._device_name
 
+    @property
+    def model(self):
```

Comment:
This is not used anywhere ?

Reply:
It's not. I thought it was accessible, but it wasn't so it's gone now.